R/job_operations.R

```diff
@@ -3,14 +3,16 @@ addJob <- function(jobId,
                    jobPreparationTask = NULL,
                    usesTaskDependencies = FALSE,
                    content = "parsed",
+                   metadata,
```

Comment:
metadata default value should be NULL if the user doesn't provide it

Reply:
fixed.